packages/flutter/test/services/flutter_version_test.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```

Comment:
I'm not sure about these tests, if you have better ideas I'm all ears.

Problems with testing this is to know the Flutter version at runtime, I need to pass this information either as Dart define to the code or I need to execute the flutter tool via process at runtime, but the tool might not be available in the integration test environment. The latter is not possible in the web environment. The former requires a lot of code to make all the information even available to an integration test, as it's not individually available to query.

Instead, I wrote more extensive tool tests and here I just verify that the properties match their respective Dart defines.